### DISCUSSION TITLE
eNyugta másolat

### DISCUSSION THREAD

**Discussion by MicraMetripond** [nav-gov-hu/eRECEIPT#16] [Kérdések és válaszok]
Hogy tudja az új eNyugta rendszer megakadályozni azt, ha egy boltos több embernek (aki ugyanazt veszi és nem azonosítja magát) nyomtatja ki ugyanazt a nyugta másolatot és közben nyugta csak egyszer szerepel a nyugtatárban? Az összes vásárló, akinek ugyanaz a QR-kód a birtokában meg tudja nézni a nyugtát a nyugtatárban?

**Comment by renced42** [comment]
Kedves @MicraMetripond 

A pénztárgépből nem lehet nyugtamásolatot csak egyszer nyomtatni a fizetést követően vásárlói kérésre. Így, ha csak az ePénztárgép nem felel meg a rendeletnek ilyen eset nem történhet meg.
Ahhoz, hogy az elektronikus nyugtát valaki el tudja olvasni szüksége van a titkosító kulcspár privát kulcsára amit csak vevői applikációban tud generálni. Persze megteheti, valaki hogy parancssorban generál kulcspárt meg QR kódot, a legfontosabb az hogy letölteni is csak regisztrált vevői appal tudja az elektronikus nyugtát.
